core/kubernetes/main/openhim/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+# OpenHIM Console Component
+
+configMapGenerator:
+  - name: core-openhim-console-configmap
+    files:
+      - ./volume/openhim-console/default.json
+
+resources:
+  # deployment
+  - ./openhim-console-deployment.yaml
+  # service
+  - ./openhim-console-service.yaml
+  # ingress
+  - ./openhim-console-ingress.yaml
```

Comment:
I think we should have a single kustomization file for the core package. The idea with packages is that they can be stood up and torn down individually. So, I'm not sure why we would want to split out the OpenHIM to be setup on it's own.

Reply:
Hmm, on second though is this done this way because we need to get the urls before we can configure the console?

Reply:
Yeah exactly. Can't have the console start before the changes have been made to it config files